edx/cc2olx
# [BD-27] [TNL-7921] [BB-3632] Hide wiki tab by default and add option to make it visible

This PR adds support of policy file and hiding of course wiki tab by default.

Also, it removes some Python 3.5 classifiers from `setup.py` and adds a few handy make commands. 

**JIRA tickets**:

- [TNL-7921](https://openedx.atlassian.net/browse/TNL-7921)
- [BB-3632](https://tasks.opencraft.com/browse/BB-3632)
- [BLENDED-749](https://openedx.atlassian.net/browse/BLENDED-749)

**Discussions**:
- [Slack thread](https://openedx.slack.com/archives/C015PT4M8AY/p1611169471002200)

**Testing instructions**:

1. Enter virtual environment.
1. `make install`.
1. `make fixture_cc`.
1. `cc2olx -i cc_fixture.imscc`.
1. Import `$(pwd)/output/cc_fixture.tar.gz` to LMS.
1. Check that course wiki tab is invisible.

**Reviewers**
- [ ] @alangsto
